install/0000_00_cluster-version-operator_01_admingate_configmap.yaml

```
@@ -8,3 +8,6 @@ metadata:
     include.release.openshift.io/hypershift: "true"
     include.release.openshift.io/ibm-cloud-managed: "true"
     include.release.openshift.io/self-managed-high-availability: "true"
+data:
+  ack-4.19-admissionregistration-v1beta1-api-removals-in-4.20: |-
+    The admissionregistration.k8s.io/v1beta1 API version is deprecated in 4.19 and will be removed in 4.20. The admissionregistration.k8s.io/v1 API versions of these resources are available for use in 4.19, and it is recommended to migrate to them before upgrading to 4.20.
```

```suggestion
    The admissionregistration.k8s.io/v1beta1 API group version is deprecated and will be removed in 4.20. Clients must instead use the corresponding resources in the admissionregistration.k8s.io/v1 group version.
```

Nit: "Group version" is an appropriate term for this. Also, I don't know what the style conventions are for these messages, but stronger phrasing seems appropriate for the recommendation. Ignoring it means breaking any client that depends on the old GV.